docs/installation.md

```diff
@@ -8,17 +8,16 @@ mainnet as either a [Full Node](./join-mainnet.md) or a
 ## Install Go
 
 Install `go` by following the [official docs](https://golang.org/doc/install).
-Remember to set your `$GOPATH` and `$PATH` environment variables, for example:
+Remember to set your `$PATH` environment variable, for example:
 
 ```bash
 mkdir -p $HOME/go/bin
-echo "export GOPATH=$HOME/go" >> ~/.bash_profile
-echo "export PATH=\$PATH:\$GOPATH/bin" >> ~/.bash_profile
+echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.bash_profile
```

Comment:
```suggestion
echo "export PATH=$PATH:$(go env GOPATH)/bin" >> ~/.bash_profile
```

Reply:
In my view, we should not teach the user how to install Go. Plus, `/usr/local/go` is not an OS-agnostic path

Reply:
I agree @alessio, we should just link to the golang install docs: https://golang.org/doc/install

Reply:
this what is being done: https://github.com/cosmos/gaia/blob/master/docs/installation.md#install-go. this bash excerpt is merely showing an example to not forget to set your env variable. we can get rid of the bash section as well if we'd like